skycultures/ruanui_sky_tahiti_and_society_islands/description.md

```diff
@@ -0,0 +1,49 @@
+# Ruanui sky : Tahiti and Society islands
```

Comment:
Please remove extra white space before “:”

Reply:
These were the small typos mentioned above. Quick to fix for us after squash-merging as well.